coffee_core/src/coffee.rs

```diff
@@ -195,10 +195,14 @@ impl CoffeeManager {
         if self.config.cln_config_path.is_none() {
             return Ok(());
         }
-        let root = self.config.cln_root.clone().unwrap();
+        let root = self.config.cln_root.clone();
+        let root =
+            root.ok_or_else(|| error!("cln root path not found, please run `coffee setup`"))?;
         let rpc = Client::new(format!("{root}/{}/lightning-rpc", self.config.network));
```

Comment:
I do not understand this, this will never fails.

Where you are checking if `cln_root` exist on the disk?